components/Nav.js

```diff
@@ -40,7 +40,7 @@ const Nav = () => {
 	useEffect(() => {
 		const onClickEv = (e) => {
 			if (mobileNavRef && !mobileNavRef?.current?.contains(e.target)) {
-				setShowMobileNav(false)
+				setShowMobileNav(true)
 			}
```

Comment:
ini kenapa di bikin true gan? bukannya ini kalo dia click selain component navnya bakal ngedismiss ya

Reply:
honestly pas gua pull dari develop button navMobilenya not working, jadi toggle navnya gabisa dibuka, ini gua cek lagi hu